test/e2e/interest-groups/index.html

```diff
@@ -4,7 +4,6 @@
 		<title>interest-groups.test</title>
 	</head>
 	<body>
-		<script src="https://unpkg.com/idb/build/iife/index-min.js"></script>
```

Comment:
new way of `dist` doesn't require this anymore

Reply:
nice, was wondering about that.